### PR TITLE
Refactor ScrollView to avoid squishing of text

### LINKED ISSUE #145: Fix design of list

### PR DESCRIPTION
**Pull request by elnaseyer**

### CODE REVIEW

File: app.config.ts [hunk 1]
```diff
@@ -41,6 +41,6 @@ export default {
   extra: {
     isDev: process.env.NODE_ENV === 'development',
     isProd: process.env.NODE_ENV !== 'development',
-    backendUrl: process.env.NODE_ENV === 'development' ? (process.env.BACKEND_URL ? process.env.BACKEND_URL : 'https://nexpo-dev.arkadtlth.se/api') : 'https://nexpo.arkadtlth.se/api',  }
-    //backendUrl: process.env.NODE_ENV === 'development' ? (process.env.BACKEND_URL ? process.env.BACKEND_URL : 'http://localhost/api') : 'http://localhost/api',  }
+    //backendUrl: process.env.NODE_ENV === 'development' ? (process.env.BACKEND_URL ? process.env.BACKEND_URL : 'https://nexpo-dev.arkadtlth.se/api') : 'https://nexpo.arkadtlth.se/api',  }
+    backendUrl: process.env.NODE_ENV === 'development' ? (process.env.BACKEND_URL ? process.env.BACKEND_URL : 'http://localhost/api') : 'http://localhost/api',  }
```
Comment (jonatan-kruse) [inline app.config.ts:45]: Change backend URL

File: screens/studentSessions/SSsListScreen.tsx [hunk 2]
```diff
@@ -62,13 +60,14 @@ export default function SSsListScreen({navigation, route}: SSsNavigation) {
     navigation.navigate(user?.role === Role.Student ? 'SSsApplicationScreen' : 'SSsApplicationsListScreen', {companyId});
   }
```
Comment (jonatan-kruse) [inline screens/studentSessions/SSsListScreen.tsx]: Bör det inte vara focusEffect här

File: screens/studentSessions/SSsListScreen.tsx [hunk 3]
```diff
@@ -62,13 +60,14 @@ export default function SSsListScreen({navigation, route}: SSsNavigation) {
...
+  React.useEffect(() => {
     setLoading(true);
     getTimeslotsAndCompany();
     setLoading(false);
-  }, []));
+  }, []);
 
   if (isLoading || company == null || user == null) {
+    getTimeslotsAndCompany();
     return(
```
Comment (jonatan-kruse) [inline screens/studentSessions/SSsListScreen.tsx]: Detta tror jag kan ge memory leeks

**Review by jonatan-kruse — CHANGES_REQUESTED** [review]
Nice that you got it working! First pull from origin main

File: app.config.ts (hunk 1 above)
Comment (elnaseyer) [reply app.config.ts:45]: näe

**Review by jonatan-kruse — APPROVED** [review]
Great work investigating and fixing!

**Pull request merged**